pkg/controller/certificaterequests/acme/acme.go

```diff
@@ -200,11 +200,22 @@ func (a *ACME) Sign(ctx context.Context, cr *v1.CertificateRequest, issuer v1.Ge
 
 // Build order. If we error here it is a terminating failure.
 func buildOrder(cr *v1.CertificateRequest, csr *x509.CertificateRequest) (*cmacme.Order, error) {
+	ipAddresses := []string{}
+	for _, ip := range csr.IPAddresses {
+		ipAddresses = append(ipAddresses, ip.String())
+	}
+
+	dnsNames := []string{}
+	if csr.DNSNames != nil {
+		dnsNames = csr.DNSNames
+	}
```

Comment:
Is this change necessary? It avoids assigning a `nil` slice to the OrderSpec, but why?

Reply:
Changed it